deploy.sh

```diff
@@ -0,0 +1,3 @@
+rm GalileoBabel.zip
+zip GalileoBabel.zip galileo_babel_s3.py 
+aws s3 cp GalileoBabel.zip s3://galileo-babel-lambda
```

Comment:
I think you need to run it each time you update the lambda, but I'm a bit confused as I didn't write it. There's also an `update.sh` script needed to update the function code from the version in the bucket, but as I said in Slack, I'm not sure if triggering a cosmos redeploy would work, and if so, is a better way to do that part.

:confused-face

Reply:
The following is what I want to say; however, I'm not convinced that it will work because of https://jira.dev.bbc.co.uk/browse/EDPLAT-619 (as I think it will result in a new lambda being created on the test environment, if we don't do something clever) so I think I'll need to think about this some more:

Right, thanks - if it's a Cosmos component, then any deployments should be done via Cosmos, not through running scripts. Another option is to manage it outside of Cosmos, through AWS Cloudformation directly (e.g. see [here](https://github.com/bbc/ugc-aws/blob/master/infrastructure/stack_templates/s3_backup_lambda.py#L67)), but that would mean disabling the Cosmos component and any associated Jenkins jobs.

Given the deploy and update scripts basically seem to do the equivalent of  [this](https://github.com/bbc/ugc-aws/blob/master/infrastructure/stack_templates/s3_backup_lambda.py#L67) I would prefer one of the following:
1. Try to deploy using Cosmos in the usual way, to see if that just "works"
2. Just manage via Cloudformation and disable the Cosmos lambda, etc

Reply:
Ok, well I was trying to get rid of the need to run scripts, and had it doing exactly [this|https://github.com/bbc/ugc-aws/blob/master/infrastructure/stack_templates/s3_backup_lambda.py#L67] - that was working fine for me until Cosmos rollled it back.

I wasn't aware of https://jira.dev.bbc.co.uk/browse/EDPLAT-619 - and also https://jira.dev.bbc.co.uk/browse/EDPLAT-611 - this explains a lot.

But let's have a conversation about this and work out the best way to resolve it.